articles/quickstart/native/windows-uwp-csharp/index.yml

```
@@ -1,4 +1,4 @@
-title: Windows Universal App C#
+title: UWP C#
```

```suggestion
title: UWP
```
Let's drop the C#, all .NET quickstarts are C# and none mention that.